packages/token-bridge-sdk/src/hooks/useArbTokenBridge.ts

```diff
@@ -95,49 +92,52 @@ export interface TokenBridgeParams {
 }
 
 export const useArbTokenBridge = (
-  params: TokenBridgeParams,
-  autoLoadCache = true
+  params: TokenBridgeParams
 ): ArbTokenBridge => {
   const { walletAddress, l1, l2 } = params
   const [bridgeTokens, setBridgeTokens] = useState<
     ContractStorage<ERC20BridgeToken> | undefined
   >(undefined)
 
+  const { tokenL2Addresses, tokenL1Addresses } = useMemo(() => {
```

Comment:
`bridgeTokens` can be `undefined` at some point (like when switching networks), should we add a check?

Reply:
Removed. But it should not have been an issue, if `bridgeToken` was undefined, there would be no iteration